frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipeline-task-runs-display.feature

```diff
@@ -78,4 +78,24 @@ Feature: Display task runs page
               And user clicks on task run "pipeline-runs-taskrun-results"
              Then user is redirected to Task Run Details tab
              When user scrolls to the Task Run results section
-             Then user can see Name and Value column under Task Run results
\ No newline at end of file
+             Then user can see Name and Value column under Task Run results
+
+        @regression
+        Scenario: Check for Duration column when enabled
+            Given user is at PipelineRuns tab with pipeline runs for pipeline "pipe-one"
+             When user clicks on Pipeline Run for "pipe-one"
+              And user clicks on TaskRuns tab
+              And user clicks manage columns button
+              And user enables Duration column
+             Then user sees Duration column
+
+        @regression
+        Scenario: Check for absence of Duration column when disabled
+            Given user is at PipelineRuns tab with pipeline runs for pipeline "pipe-one"
+             When user clicks on Pipeline Run for "pipe-one"
+              And user clicks on TaskRuns tab
+              And user clicks manage columns button
+              And user enables Duration column
```

Comment:
Duration column has been already enabled here, so no need to check duration option again, instead we can check for duration option already checked.